tests/integration/test_interaction_assessment.py

```diff
@@ -130,6 +130,35 @@ def test_misplaced_items_returned_to_bank(self):
         for item_id in misplaced_items:
             self.assert_reverted_item(item_id)
 
+    def test_misplaced_items_not_returned_to_bank_on_final_attempt(self):
+        """
+        Test items placed to incorrect zones are not returned to item bank
+        after submitting solution on the final attempt
```

Comment:
@arbrandes This test also checks what happens when refreshing the page; it would be worth mentioning that here I think.